vaadin-component-demo.html

```diff
@@ -58,64 +60,22 @@
 <dom-module id="vaadin-component-demo">
   <template>
     <style>
-       :host {
+      :host {
         display: block;
         max-width: 60em;
         margin-left: auto;
         margin-right: auto;
       }
-
-      nav {
-        box-shadow: inset 0 -1px 0 0 rgba(0, 0, 0, 0.1);
-        margin-bottom: 3em;
-        overflow: auto;
-        -webkit-overflow-scrolling: touch;
-      }
-
-      nav ul {
-        padding: 0;
-        margin: 0;
-        display: flex;
-      }
-
-      nav ul li {
-        list-style: none;
-        flex: none;
-      }
-
-      nav a {
-        display: inline-block;
-        padding: 1em;
-      }
-
-      nav a {
-        color: #829191;
-        text-decoration: none;
-        border-bottom: 2px solid transparent;
-      }
-
-      nav a:focus,
-      nav a:hover,
-      nav a[active] {
-        color: #2d3033;
-      }
-
-      nav a[active] {
-        border-color: #44d62c;
-      }
-
     </style>
 
     <app-location id="appLocation" route="{{route}}" use-hash-as-path="[[devMode]]"></app-location>
     <app-route id="appRoute" route="{{route}}" pattern="[[routePattern]]:page" data="{{routeData}}"></app-route>
 
-    <nav id="nav">
-      <ul>
-        <template is="dom-repeat" items="[[_demoConfig.pages]]" as="page" id="navigationRepeat">
-          <li><a href$="[[linkBase]][[page.url]]" name$="[[page.url]]">[[page.name]]</a></li>
-        </template>
-      </ul>
-    </nav>
+    <vaadin-tabs selected="{{selectedPageIndex}}">
+      <template is="dom-repeat" items="[[_demoConfig.pages]]" as="page" id="navigationRepeat">
+        <vaadin-tab>[[page.name]]</vaadin-tab>
```

Comment:
@limonte Please rollback the link removal. We need the link to be able to e.g. open it in new tab by pressing Cmd+click etc, and it is impossible with current workflow.

Note you will also need some styles to make link expand to full tab, see https://github.com/web-padawan/elements-viewer/blob/2373758938ea7c584dbb0dba0a49b2f80ed7f116/src/viewer-link.html#L13